c2corg_ui/templates/outing/associated.html

```diff
@@ -3,7 +3,7 @@
   <article class="associated-documents" >
     <div class="list-item outing" ng-repeat="outing in document['associations']['recent_outings']['outings']">
       <%include file="../outing/card.html"/>
-      <app-delete-association parent-id="document.document_id" child-id="outing.document_id" ng-if="userCtrl.auth.isAuthenticated()"></app-delete-association>
+      <span class="glyphicon glyphicon-trash" ng-click="editCtrl.removeAssociation(outing.document_id, 'outings')" ng-if="userCtrl.auth.isAuthenticated()"></span>
```

Comment:
I mean: if we no longer need the ``app-delete-association`` directive and if it is no longer used anywhere else, should we completely remove the directive file. It is still available at https://github.com/c2corg/v6_ui/blob/editing-fixes/c2corg_ui/static/js/deleteassociationdirective.js